pkg/validation/internal/community.go

```diff
@@ -24,6 +24,9 @@ const ocpLabelindex = "com.redhat.openshift.versions"
 // OCP version where the apis v1beta1 is no longer supported
 const ocpVerV1beta1Unsupported = "4.9"
 
+// olm.properties is a CSV annotation that defines the properties of a bundle.
+const olmproperties = "olm.properties"
```

Comment:
this seems like an unrelated gofmt change? does this need to be included in the commit?

Reply:
So this was actually done by me so that I could run this if statement against a constant, instead.

```
if annotationKey == olmproperties {
```